chapters/hpmor-chapter-035.tex

```diff
@@ -231,7 +233,7 @@
 
 Professor McGonagall pressed her hands over her eyes. When she took them away, her lined face looked very old. “Yes,” she said in a whisper, “you would have done well in my House. Stay safe, Miss~Granger, and be careful. And if you are ever worried or uncomfortable about anything, please come to me at once. I won’t keep you any longer.”
 
-\latersection{Aftermath, Draco Malfoy:}
+\latersection{Aftermath: Draco Malfoy.}
```

Comment:
Remove full stop

Reply:
done (I think I have this in another PR, but now fixed here nevertheless ;-)